canonical/global-nav
# Add Anbox Cloud to product list

## Done
- Added new entry for Anbox Cloud to product list

## QA
- Go to https://global-nav-266.demos.haus/
- Open the navigation
- Check the new entry for Anbox Cloud

## Screenshot

![image](https://github.com/canonical/global-nav/assets/56583786/6d0e6bf6-7a91-43fc-bf04-b8374ad7d993)
